mrmustard/physics/gaussian_integrals.py

```diff
@@ -472,3 +392,112 @@ def contract_two_Abc_poly(
     return complex_gaussian_integral(
         Abc, idx1, tuple(n + Abc1[0].shape[-1] - dim_n1 for n in idx2), measure=-1.0
     )
+
+
+def complex_gaussian_integral_2(
```

Comment:
Doing a ctrl + F it doesnt seem like we use complex_gaussian_integral_2 in MrMustard. Are we planning on using it or is this something used in Mr Kite primarily?

Reply:
in MM. I just need to plug it where needed.